Casks/a/ableset.rb

```diff
@@ -0,0 +1,33 @@
+cask "ableset" do
+  arch arm: "aarch64", intel: "x86_64"
+  folder = on_arch_conditional arm: "-arm64", intel: ""
+
+  version "2.7.1"
+  sha256 arm:   "27e7bd8d55b474c92589b79c6272269151a9941de82969698338d9c2d332a3a7",
+         intel: "6c312f3b5cf08943fd837eb8a5f81d8fc0e8ede592bb99c18b423f7a400ce084"
+
+  url "https://ableset.app/download/mac#{folder}/#{version}"
+  name "AbleSet"
+  desc "Ableton setlist manager"
+  homepage "https://ableset.app/"
+
+  livecheck do
+    url "https://ableset.app/download/mac/"
+    strategy :header_match
```

Comment:
```suggestion
    url "https://s3.amazonaws.com/download.ableset.app/mac#{arch}/latest-mac.yml"
    strategy :electron_builder
```

We can use the feed of the in-app updater with the `ElectronBuilder` strategy.

Reply:
Changed in latest commit